sample_files/.travis.yml

```diff
@@ -25,11 +23,6 @@ addons:
 #       https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise
 #      - wkhtmltopdf  # only add if needed and check the before_install section below
 
-# set up an X server to run wkhtmltopdf.
```

Comment:
Is this not needed anymore?

Reply:
fortunately not with the patched version. Check the PR above, now the build is finally green with this :tada: